app/admin-tab/processes/process/template.hbs

```diff
@@ -4,6 +4,6 @@
 {{#each model.processExecutions as |process|}}
   <section class="well">
     <h4>Child Execution: {{process.id}}</h4>
-    {{process-executions execution=process.log.executions}}
+    {{process-executions execution=process.log.executions apiView=model.processExecutions.links.self applicationController=application}}
```

Comment:
Application is auto-injected into all components with `app/initializers/inject-application'